pmm/pmm2-release.groovy

```diff
@@ -551,8 +543,33 @@ ENDSSH
             unstash 'copy'
             script {
                 def IMAGE = sh(returnStdout: true, script: "cat copy.list").trim()
-                slackSend botUser: true, channel: '#pmm-ci', color: '#00FF00', message: "[${specName}]: build finished - ${IMAGE}"
+                slackSend botUser: true,
+                          channel: '#pmm-ci',
+                          color: '#00FF00',
+                          message: "[${specName}]: build finished - ${IMAGE}"
+                slackSend botUser: true,
+                          channel: '#releases',
+                          color: '#00FF00',
+                          message: "PMM ${VERSION} was released!"
```

Comment:
how we can say that it was released as you haven't tested it? maybe repo is broken?